package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "sfcc-devtools",
+  "version": "1.0.0",
+  "description": "Support Browser Interaction with VS Code, Eclipse, and SFCC Business Manager via DevTools Web Inspector.",
+  "author": "Red van Workshop",
+  "license": "MIT",
+  "private": false,
```

Consider adding a repository property
```suggestion
  "private": false,
  "repository": "github:redvanworkshop/sfcc-devtools",
```

---

Good call, I will copy over what we are using in our other repos